gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py

```diff
@@ -168,6 +168,7 @@ def fake_sa_req_list_fw(self):
                                 ],
                                 "description": '{\
                                     "vm_management_ip": "172.24.4.5",\
+                                    "provider_cidr": "11.0.1.0/24",\
```

Comment:
This won't. I ran fw test cases. There are some which are failing but not bec of this reason.